medusa/server/api/v2/base.py

```diff
@@ -319,7 +319,7 @@ def _get_page(self):
         except ValueError:
             self._raise_bad_request_error('Invalid page parameter')
 
-    def _get_limit(self, default=20, maximum=1000):
+    def _get_limit(self, default=20, maximum=10000):
```

Comment:
@sharkykh the one problem with this approach that we're now skipping pagination. Maybe we should open an issue about this so we don't forget?